arcgis-ios-sdk-samples/Scenes/Show labels on layer in 3D/ShowLabelsOnLayer3DViewController.swift

```diff
@@ -33,9 +33,8 @@ class ShowLabelsOnLayer3DViewController: UIViewController {
         // Load the scene.
         scene.load { [weak self] error in
             guard let self = self else { return }
-            // Get the feature layer.
-            if let operationalLayers = scene.operationalLayers as? [AGSGroupLayer],
-               let groupLayer = operationalLayers.first(where: { $0.name == "Gas" }),
+            // Get the "Gas Main" feature layer.
+            if let groupLayer = scene.operationalLayers.compactMap({ $0 as? AGSGroupLayer }).first(where: { $0.name == "Gas" }),
                let layers = groupLayer.layers as? [AGSLayer],
```

Comment:
Does the scene have many operational layers? If not, then what you have is fine. But if it does, it would be good to lazily compact map, so as not to _map_ all layers and then find the first one (which is the case now):

```swift
if let groupLayer = scene.operationalLayers.lazy.compactMap({ $0 as? AGSGroupLayer }).first(where: { $0.name == "Gas" }),
```

Another way is to just use `first(where:)`, although that does require casting the found layer twice:

```swift
if let groupLayer = scene.operationalLayers.first(where: { ($0 as? AGSGroupLayer)?.name == "Gas" }) as? AGSGroupLayer,
```

Reply:
This scene has 7 layers. I'll update it to lazily compact!